src/Core/Commander/ManagerCommands.js

```diff
@@ -40,6 +40,9 @@ define('Core/Commander/ManagerCommands', [
             this.providerMap = {};
             this.history = null;
             this.eventsManager = new EventsManager();
+            this.currentCommandsCount = 0;
+            this.maxConcurrentCommands = 16;
```

Comment:
`requestIdleCallback`? https://twitter.com/jacobrossi/status/667047851415969792 & http://permalink.gmane.org/gmane.org.w3c.webperf/757 (not more standard, but with public support from several vendors: https://www.chromestatus.com/feature/5572795866021888 / https://platform-status.mozilla.org/#request-idle-callback)

FWIW, that `tick` function is the one I described above, and more or less what can be found in Nicholas Zakas' blog entry I linked to ;-)